README.md

```diff
@@ -49,10 +49,10 @@ tasks:
 
 ## Variables
 
-- `backports_uri`: URI of the backports repository; change this if you want to use a particular mirror.
+- `backports_internal_uri`: URI of the backports repository; change this if you want to use a particular mirror.
   - Debian: `http://deb.debian.org/debian`
   - Ubuntu: `https://archive.ubuntu.com/ubuntu`
-- `backports_components`: Release and components for sources.list
+- `backports_internal_components`: Release and components for sources.list
```

Comment:
`backports_internal_uri` and `backports_internal_components` should not be mentioned in README since they are, well, internal. Same for MIGRATING file.

Reply:
I rebased it to master and changed your  remarks